src/components/WeekDayPicker/readme.md

```diff
@@ -0,0 +1,175 @@
+##### WeekDayPicker base:
+
+```js
+import React, { useState } from 'react';
+import { WeekDayPicker } from 'react-rainbow-components';
+
+const SimpleWeekDayPicker = (props) => {
+    const [day, setDay] = useState('');
+
+    const handleOnChange = e => {
+        const weekDay = e.currentTarget.value;
+        setDay(weekDay);
+        console.log(weekDay);
+    };
+
+    return (
+        <WeekDayPicker 
+            label="Select a day" 
+            value={day}
+            onChange={(e) => handleOnChange(e)}
+        /> 
+    );
+};
+
+<div className="rainbow-align-content_center rainbow-m-around_xx-large">
+    <SimpleWeekDayPicker  />
+</div>
+```
+
+##### WeekDayPicker multiple selection:
+
+```js
+import React, { useState, useEffect } from 'react';
+import { WeekDayPicker } from 'react-rainbow-components';
+
+const MultipleWeekDayPicker = (props) => {
+    const [days, setDays] = useState([]);
+
+    useEffect( () => {
+        console.log(days);
+    }, [days])
+
+    const handleOnChange = e => {
+        const weekDay = e.currentTarget.value;
+        const isChecked = e.currentTarget.checked;
+
+        if (isChecked && !days.includes(weekDay)) {
+            setDays([...days, weekDay]);
+        } else {
+            setDays(days.filter(day => day !== weekDay))
+        }
```

Comment:
same ^^ this logic should be implemented in the component index.js since we don't want the users to have the need to reimplement each time they used the component